apps/backend/src/modules/users/user.repository.ts

```diff
@@ -28,8 +32,15 @@ class UserRepository implements Repository {
 		return Promise.resolve(true);
 	}
 
-	public find(): ReturnType<Repository["find"]> {
-		return Promise.resolve(null);
+	public async find({
+		attribute,
+		value,
+	}: {
+		attribute: UserAttribute;
+		value: Objection.PrimitiveValue;
+	}): Promise<null | UserEntity> {
```

Comment:
`find` method should only search by id. To search by other fields add new methods, such as `findByEmail`